contribute/style-guides/storybook.md

```diff
@@ -0,0 +1,175 @@
+# Why creating stories
+
+[Storybook](https://storybook.js.org/) is a tool which we use to manage our design system and the components which are a part of it. Storybook consists of stories: each story representing a component and a case in which it is used. To show a wide variety of use cases is good both documentation wise and for troubleshooting - it might be possible to reproduce a bug for an edge case in a story.
+
+In short:
+
+- A good way to publish our design system with its implementations
+- Used as a tool for documentation
+- Used for debugging and displaying edge cases
+
+# How to create stories
+
+Stories for a component should be placed next to the component file. The Storybook file requires the same name as the component file. For example, a story for `SomeComponent.tsx` will have the file name `SomeComponent.story.tsx`.
+
+## Writing stories
+
+When writing stories we use the [CSF format](https://storybook.js.org/docs/formats/component-story-format/). For more in depth information on writing stories see [Storybook’s documentation on writing stories](https://storybook.js.org/docs/basics/writing-stories/).
+
+With the CSF format the default export defines some general information about the stories in the file
+
+- `title`: Where the component is going to live in the hierarchy
+- `decorators`: a list which can contain wrappers or provide context, such as theming
+
+```jsx
+// In MyComponent.story.tsx
+
+import MyComponent from './MyComponent';
+
+export default {
+  title: 'General/MyComponent',
+  component: MyComponent,
+  decorators: [ ... ],
+}
+
+```
+
+When it comes to writing the actual stories, you continue in the same file with named exports - which are turned into the story name.
+
+```jsx
+// Will produce a story name “some story”
+export const someStory = () => <MyComponent />;
+```
+
+If you want to write cover cases with different values for props you don’t need to create a new story - using Knobs is usually enough. This will be covered further down.
+
+## Categorization
+
+We currently have these categories:
+
+- Docs Overview - Guidelines and information regarding the design system
+- Forms - Components commonly used in forms such as different kind of inputs
+- General - Components which can be used in a lot of different places
+- Visualizations - Data visualizations
+- Panel - Components belonging to panels and panel editors
+
+# Writing MDX documentation
```

Comment:
Please add a short intro after this heading to provide context for the information in this section.

Also, it is a best practice to only have one h1 per topic, which is the h1 at the top of this page. Please change this to h2 and adjust the subheadings accordingly.